src/Spatial/Euclidean/PolyLine3D.cs

```diff
@@ -10,58 +10,53 @@
     /// </summary>
     public class PolyLine3D : IEnumerable<Point3D>
     {
+        /// <summary>
+        /// An internal list of points
+        /// </summary>
         private readonly List<Point3D> points;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PolyLine3D"/> class.
+        /// Creates a PolyLine3D from a pre-existing IEnumerable of Point3Ds
+        /// </summary>
+        /// <param name="points">A list of points.</param>
         public PolyLine3D(IEnumerable<Point3D> points)
         {
```

Comment:
We could add a null check here, throwexpression perhaps. Just a reminder, no need to fix in this PR.

Reply:
this is just the doco pr, haven't yet started tuning the class